src/components/Text/Text.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../assets/variables.scss";
+
+.text {
+  font-weight: $semibold;
+  color: $black;
+  
+
+  &_h1 {
+    font-family: "Montserrat";
```

Comment:
`font-family` is duplicated for every class - wouldn't it be nicer to put it in `.text` and assign that class to the component along the type based on class (`text-h1`, `text-h2`, ...)